solvation_analysis/analysis.py

```diff
@@ -0,0 +1,103 @@
+# from abc import ABC
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.analysis.rdf import InterRDF
+import numpy as np
+
+
+class Solute:
+    def __init__(self, solute, solvents, **kwargs):
+        super(Solute, self).__init__(solute.universe.trajectory, **kwargs)
+        self.solute = solute
+        self.solvents = solvents
+
+    def run(self):
+        # run all RDFs and store their results
+        rdfs = []
+        for solvent in self.solvents:
+            rdf = InterRDF(self.solute, solvent)
+            rdf.run()
+            rdfs.append(rdf)
+        self.rdfs = rdfs
+        self.ion_speciation = IonSpeciation(self.solute, self.solvents)
+        self.ion_pairing = IonPairing(self.solute, self.solvents)
+        self.coordination_numbers = CoordinationNumber(self.solute, self.solvents)
+
+
+class IonSpeciation(AnalysisBase):
```

Comment:
If you can fill out the docstring, I will have a better idea of what functionality this will encompass.

Reply:
I am going to delete these classes soon and I will resolve these issues when I do.